examples/reference/panes/Perspective.ipynb

```diff
@@ -17,82 +21,250 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The ``Perspective`` pane provides an interactive visualization component for large, real-time datasets built on the [Perspective](https://perspective.finos.org/) project.\n",
+    "The ``Perspective`` pane provides a powerful visualization component for large, real-time datasets built on the [Perspective Project](https://perspective.finos.org/). **`Perspective` brings *excel-like* capabilities to your data app**. Check out the [Perspective Examples Gallery](https://perspective.finos.org/examples/) for inspiration.\n",
+    "\n",
+    "The ``Perspective`` pane is a very good alternative to the [`Tabulator`](../widgets/Tabulator.ipynb) widget.\n",
```

Comment:
Cannot verify that this link will work in Sphinx docs. Will it @philippjfr ?

Reply:
Yes, there was a bug in the pyodide renderer that didn't enforce that perspective was loaded before attempting to render the output.